cookiecutter/utils.py

```diff
@@ -110,3 +110,41 @@ def prompt_and_delete(path, no_input=False):
             return False
 
         sys.exit()
+
+
+# Modified from python 3.7.1
+def copytree(src, dst, symlinks=False, exist_ok=False):
+    names = os.listdir(src)
+
+    os.makedirs(dst, exist_ok=exist_ok)
+    errors = []
+    for name in names:
+        srcname = os.path.join(src, name)
+        dstname = os.path.join(dst, name)
+        try:
+            if os.path.islink(srcname):
+                linkto = os.readlink(srcname)
+                if os.path.isdir(srcname):
+                    copytree(srcname, dstname, symlinks)
```

Comment:
Recursive calls to `copytree` should respect the `exist_ok` value that was provided:

`copytree(srcname, dstname, symlinks, exist_ok)`